docs/guides/installation.md

```diff
@@ -512,6 +512,10 @@ flux create helmrelease nginx \
 
 ## Upgrade
 
+!!! note "Versions"
+Any PATCH version can be used to upgrade to a new MINOR version. It is for
```

Comment:
I would encourage the user of the latest patch release, something like:

Please use the latest PATCH version when upgrade to a new MINOR version.

Reply:
Changed the wordings a bit.